tests/automation/linked_device_user.spec.ts

```diff
@@ -423,4 +431,68 @@ test_Alice_2W_Bob_1W(
       ),
     ]);
   },
-);
\ No newline at end of file
+);
+
+test_Alice_2W(
+  'Hide note to self syncs',
+  async ({ alice, aliceWindow1, aliceWindow2 }) => {
+    await clickOnTestIdWithText(aliceWindow1, 'new-conversation-button');
+    await clickOnTestIdWithText(
+      aliceWindow1,
```

Comment:
`Hide Note to Self` and `Note to Self` should probably use the localized string, right?